consensus/istanbul/backend/engine.go

```diff
@@ -195,8 +195,15 @@ func (sb *Backend) VerifyHeaders(chain consensus.ChainReader, headers []*types.H
 	} else {
 		go func() {
 			err := sb.batchVerifyHeaders(chain, headers)
-			// only 1 result will be returned by the channel
-			results <- err
+			// even though we get 1 return value from the batched verification
```

Comment:
So I was not too happy with this change, but it seems like there's no good alternative? In the `if` branch we have an array of results due to iterating over the headers, but in the `else` branch, since we're doing _batch_ verification there is just 1 result _for all headers_. I chose to just feed that result in all slots in the channel. Happy to discuss alternatives (not sure if there are any, without significant changes in the interfaces)